tensorflow/core/kernels/list_kernels.cc

```diff
@@ -179,7 +179,7 @@ Status GetInputList(OpKernelContext* c, int index, const TensorList** list) {
                                    c->input(index).shape().DebugString());
   }
   const TensorList* l = c->input(index).scalar<Variant>()().get<TensorList>();
-  if (l == nullptr) {
+  if (l == nullptr) { 
```

Comment:
Why was a space added here?

Reply:
It was added by my kitty :>

---
In reply to: [514684656](https://github.com/microsoft/tensorflow-directml/pull/78#discussion_r514684656) [](ancestors = 514684656)

Reply:
:smile_cat: